packages/hardhat/config/config.dev.json

```diff
@@ -21,6 +22,7 @@
       "USDC_Address": "0x2791Bca1f2de4661ED88A30C99A7a9449Aa84174",
       "XToken_Address": "0x7ceB23fD6bC0adD59E62ac25578270cFf1b9f619",
       "CGT_Holder": "0x032c079DB811541501Bd769836490Ce0f5999264",
+      "USDC_Holder": "0xf89d7b9c864f589bbf53a82105107622b35eaa40",
```

Comment:
Isnt Rinkeby deprecated can we switch to sepolia

Reply:
Yes, so I've added Goerli, since i found it be more favorable due to its support for both the UniswapV3 SwapRouter contract(for the wrapper contract) and the Chainlink price feed.